example/ios/Supporting Files/build.gradle.kts

```diff
@@ -13,16 +12,26 @@ buildscript {
         classpath("com.android.tools.build:gradle:$android_gradle_plugin_version")
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
         classpath("org.jetbrains.kotlin:kotlin-serialization:$kotlin_version")
-        classpath("com.google.gms:google-services:4.3.5")
-        classpath("org.jlleitschuh.gradle:ktlint-gradle:10.0.0")
+        classpath("com.google.gms:google-services:4.3.10")
```

Comment:
Can we move these versions to ext.gradle so its all in one place

Reply:
Ad far as I know, you can't.

Reply:
The reason is `buildscript` runs before everything else (before `ext` is included) and is very restricted (can't include other files for example).

I've moved these to `local.properties` and `settings.gradle.kts` which are already kept in sync with each other. I've also simplified the settings file quite a bit.